components/core/marketing/Issue.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from "react";
+import * as Constants from "~/common/constants";
+
+import { css } from "@emotion/react";
+
+const STYLES_SLATE_CARD_GROUP = css`
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+  margin-top: 48px;
+`;
+const STYLES_SLATE_CARD = css`
```

Comment:
Formatting, and the name of these `STYLES_` don't make sense. Right?

Reply:
Got it misunderstood the previous pr comment.